src/components/listing-page-components/Overview/index.jsx

```diff
@@ -0,0 +1,101 @@
+import Button from 'components/shared/Button';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Claim,
+  FormContainer,
+  MainContainer,
+  OverviewClaimScheduleContainer,
+  OverviewParagraph,
+  Partition,
+  ScheduleVisit,
+  Statistics,
+  Title,
+  Value,
+} from './overview.styles';
+
+/**
+ * Listing page overview section.
+ *
+ * @param {Object} props The component props.
+ * @param {Object} props.statistics Post statistics.
+ * @param {String[]} props.overview Asset's overview paragraph.
+ *
+ * @return {JSX.Element}
+ */
+function Overview({ statistics, overview }) {
```

Comment:
less props that serve one goal = easier to read and use the component
```suggestion
function Overview({data }) {
```